### DISCUSSION TITLE
Providing a roadmap/planning for the future

### DISCUSSION THREAD

**Discussion by FSharpCSharp** [WerWolv/ImHex#574] [General]
I have been following this exciting project for a few months now. There is, after all, active development taking place. What I personally still miss is a roadmap, or an intensified use of the project features available on GitHub. This would give a much better overview of upcoming features.

Would it be possible to provide something like that in the future? That would be really desirable. Maybe this already exists, only I overlooked it, in which case it would be great if you could give me a link to it.

**Comment by WerWolv** [comment]
Hey! 
When I started ImHex, I was using the Projects feature to try and document things a bit. However I quickly noticed that it usually was really out of date within just a few days.
The problem is, that I don't exactly know myself what I'm going to add in the future. I often just get random ideas on a new feature and then I sit down and implement it right away. 

Right now there are two bigger things planned which is the rewrite of the Diff View and the Project File system but that's basically it.
I really don't know what to put on the Projects page tbh
